db/migrate/20170919211745_create_vm_subnets.rb

```diff
@@ -0,0 +1,11 @@
+class CreateVmSubnets < ActiveRecord::Migration[5.0]
+  def change
+    create_table :vm_subnets do |t|
```

Comment:
@agrare should we reuse our cloud_subnet model? Ignoring the bad naming for now...

Reply:
@agrare @Ladas I think that would result in more confusion, especially as that table seems to expect relationships to other cloud_xxx tables.

Reply:
right, so best to get together with @Fryguy, since there seems to be some architecture shift. Not sure if we should use different Model for very similar things (AFAIK subnet is just subnet)